README.md

```diff
@@ -1,33 +1,38 @@
-Colours in terminal
-==================
-It's a common confusion about terminal colours... Actually we have this:
-* plain ascii
-* ansi escape codes (16 colour codes with bold/italic and background)
-* 256 colour palette (216 colours + 16 ansi + 24 gray) (colors are 24bit)
-* 24bit true colour ("888" colours (aka 16 milion))
+# Terminal Colours
+
+There exists common confusion about terminal colours. This is what we have right now:
+
+- Plain ASCII
+- ANSI escape codes: 16 colour codes with bold/italic and background
+- 256 colour palette: 216 colours + 16 ANSI + 24 gray (colors are 24-bit)
+- 24-bit true colour: "888" colours (aka 16 milion)
 
 ```
 printf "\x1b[${bg};2;${red};${green};${blue}m\n"
 ```
 
-The 256 colour palete is configured at start, and it's a 666 cube of
-colours, each of them defined as a 24bit (888 rgb) colour.
+The 256-colour palette is configured at start and is a 666-cube of colours,
+each of them defined as a 24-bit (888 rgb) colour.
 
-This means that current support can only display 256 different colours
-in the terminal, while truecolour means that you can display 16 milion
-different colours at the same time.
+This means that current support can only display 256 different colours in the
+terminal while "TrueColour" means that you can display 16 million different
```

Comment:
I believe here it should be space in "'true colour"

Reply:
Fixed, and also fixed #5. 

I think that `truecolor` the $COLORTERM setting should be one word, but when referring to the concept of 24bit color, use two words like "true color". I have made this change as part of the commit.